core/vm/evm.go

```diff
@@ -47,14 +47,33 @@ type (
 
 // run runs the given contract and takes care of running precompiles with a fallback to the byte code interpreter.
 func run(evm *EVM, contract *Contract, input []byte, readOnly bool) ([]byte, error) {
-	// Intercept the StateManager calls
-	if contract.Address() == StateManagerAddress {
-		log.Debug("Calling State Manager contract.", "StateManagerAddress", StateManagerAddress.Hex())
-		ret, err := callStateManager(input, evm, contract)
-		if err != nil {
-			log.Error("State manager error!", "error", err)
+	if UsingOVM {
+		// OVM_ENABLED
+		var isUnknown = true
+		for name, account := range OvmStateDump.Accounts {
+			if contract.Address() == account.Address {
+				isUnknown = false
+				abi := &(account.ABI)
+				method, err := abi.MethodById(input)
+				if err != nil {
+					log.Debug("Calling Known Contract", "Name", name)
+				} else {
+					log.Debug("Calling Known Contract", "Name", name, "Method", method.RawName)
+				}
+			}
+		}
+
+		if isUnknown {
+			log.Debug("Calling Unknown Contract", "Address", contract.Address().Hex())
+		}
+
+		if contract.Address() == OvmStateDump.Accounts["OVM_SafetyChecker"].Address {
+			return common.FromHex("0x0000000000000000000000000000000000000000000000000000000000000001"), nil
```

Comment:
Temporary: have the safety checker always return true.

Reply:
This would be a good comment to leave directly in the code